kivymd/vendor/circularTimePicker/__init__.py

```diff
@@ -171,8 +171,7 @@ def rgb_to_hex(*color):
 
 
 class Number(Label):
-    """The class used to show the numbers in the selector.
-    """
+    """The class used to show the numbers in the selector."""
 
     size_factor = NumericProperty(0.5)
     """Font size scale.
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a